apps/example-orchestrator-ui/components/Subscription/Subscription.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { FC, useMemo, useState } from 'react';
+import {
+    ProcessesTimeline,
+    SubscriptionActions,
+    SubscriptionDetailTree,
+    SubscriptionGeneral,
+} from '@orchestrator-ui/orchestrator-ui-components';
+import {
+    EuiBadge,
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiTab,
+    EuiTabs,
+    EuiText,
+} from '@elastic/eui';
+import { useQuery } from 'react-query';
+import { GRAPHQL_ENDPOINT } from '../../constants';
+import { GraphQLClient } from 'graphql-request';
+import {
+    GET_SUBSCRIPTION_DETAIL_COMPLETE,
+    GET_SUBSCRIPTION_DETAIL_OUTLINE,
+    mapApiResponseToSubscriptionDetail,
+} from './subscriptionQuery';
+import { SubscriptionContext } from '@orchestrator-ui/orchestrator-ui-components';
+import { getColor, tabs } from './utils';
+
+type SubscriptionProps = {
+    subscriptionId: string;
+};
+
+const graphQLClient = new GraphQLClient(GRAPHQL_ENDPOINT);
+
+export const Subscription: FC<SubscriptionProps> = ({ subscriptionId }) => {
```

Comment:
`Subscription` component needs to be covered by both flavours of UI integration test(s) namely Jest snapshot tests and Storybook stories towards avoiding unintentional regressions both in the rendered HTML and the UI itself respectively.

Reply:
I am not sure if that's viable at the moment: Storybook functionality is currently not finished. The work in #82 is still very fresh. Probably better to start using storybook, when this is stable enough. E.g. create a ticket to cover this with tests. The project is still very young.